### ISSUE TITLE
Install Dependencies

### ISSUE THREAD

**Issue by genaiwithsudlak** [genaiwithsudlak/Customer_360_Analytics#8]
agate==1.9.1
aiohappyeyeballs==2.6.1
aiohttp==3.13.3
aiohttp-retry==2.9.1
aiosignal==1.4.0
alembic==1.18.4
altair==4.2.2
amqp==5.3.1
annotated-doc==0.0.4
annotated-types==0.7.0
antlr4-python3-runtime==4.9.3
anyio==4.12.1
appdirs==1.4.4
asyncssh==2.22.0
atpublic==7.0.0
attrs==25.4.0
autograd==1.8.0
babel==2.18.0
billiard==4.2.4
blinker==1.9.0
cachetools==6.2.6
celery==5.6.2
certifi==2026.1.4
cffi==2.0.0
charset-normalizer==3.4.4
click==8.3.1
click-didyoumean==0.3.1
click-plugins==1.1.1.2
click-repl==0.3.0
cloudpickle==3.1.2
colorama==0.4.6
colorlog==6.10.1
configobj==5.0.9
contourpy==1.3.3
cryptography==46.0.5
cycler==0.12.1
dacite==1.9.2
daff==1.4.2
databricks-sdk==0.90.0
dbt-adapters==1.22.6
dbt-common==1.37.2
dbt-core==1.11.6
dbt-duckdb==1.10.1
dbt-extractor==0.6.0
dbt-protos==1.0.431
dbt-semantic-interfaces==0.9.0
deepdiff==8.6.1
deprecation==2.1.0
dictdiffer==0.9.0
dill==0.4.1
diskcache==5.6.3
distro==1.9.0
docker==7.1.0
dpath==2.2.0
duckdb==1.4.4
dulwich==1.1.0
dvc==3.66.1
dvc-data==3.18.2
dvc-http==2.32.0
dvc-objects==5.2.0
dvc-render==1.0.2
dvc-studio-client==0.22.0
dvc-task==0.40.2
dynaconf==3.2.12
entrypoints==0.4
evidently==0.7.20
Faker==40.4.0
fastapi==0.129.0
filelock==3.24.2
filetype==1.2.0
Flask==3.1.2
flask-cors==6.0.2
flatten-dict==0.4.2
flufl.lock==9.0.0
fonttools==4.61.1
frozenlist==1.8.0
fsspec==2026.2.0
funcy==2.0
gitdb==4.0.12
GitPython==3.1.46
google-auth==2.48.0
grandalf==0.8
graphene==3.4.3
graphql-core==3.2.7
graphql-relay==3.2.0
great_expectations==1.12.3
greenlet==3.3.1
gto==1.9.0
h11==0.16.0
httpcore==1.0.9
httptools==0.7.1
httpx==0.28.1
huey==2.6.0
hydra-core==1.3.2
idna==3.11
ImageHash==4.3.2
importlib_metadata==8.7.1
iniconfig==2.3.0
isodate==0.7.2
iterative-telemetry==0.0.10
itsdangerous==2.2.0
Jinja2==3.1.6
joblib==1.5.3
jsonschema==4.26.0
jsonschema-specifications==2025.9.1
kiwisolver==1.4.9
kombu==5.6.2
leather==0.4.1
Lifetimes==0.11.3
lightgbm==4.6.0
litestar==2.21.0
litestar-htmx==0.5.0
llvmlite==0.45.1
Mako==1.3.10
markdown-it-py==4.0.0
MarkupSafe==3.0.3
marshmallow==3.26.2
mashumaro==3.14
matplotlib==3.10.0
mdurl==0.1.2
minify_html==0.18.1
mistune==3.2.0
mlflow==3.9.0
mlflow-skinny==3.9.0
mlflow-tracing==3.9.0
more-itertools==10.8.0
mpmath==1.3.0
msgpack==1.1.2
msgspec==0.20.0
multidict==6.7.1
multimethod==1.12
multipart==1.3.0
mypy_extensions==1.1.0
narwhals==2.16.0
networkx==3.6.1
nltk==3.9.2
numba==0.62.1
numpy==2.3.5
omegaconf==2.3.0
opentelemetry-api==1.39.1
opentelemetry-proto==1.39.1
opentelemetry-sdk==1.39.1
opentelemetry-semantic-conventions==0.60b1
optuna==4.7.0
orderly-set==5.5.0
orjson==3.11.7
packaging==25.0
pandas==2.3.3
parsedatetime==2.6
pathspec==0.12.1
patsy==1.0.2
phik==0.12.5
pillow==12.1.1
platformdirs==4.9.2
plotly==5.24.1
pluggy==1.6.0
polyfactory==3.2.0
prettytable==3.17.0
prompt_toolkit==3.0.52
propcache==0.4.1
protobuf==6.33.5
psutil==7.2.2
puremagic==1.30
pyarrow==22.0.0
pyasn1==0.6.2
pyasn1_modules==0.4.2
pycparser==3.0
pydantic==2.12.5
pydantic-settings==2.13.0
pydantic_core==2.41.5
pydeck==0.9.1
pydot==4.0.1
pygit2==1.19.1
Pygments==2.19.2
pygtrie==2.5.0
pyparsing==3.3.2
pytest==9.0.2
python-dateutil==2.9.0.post0
python-dotenv==1.2.1
python-slugify==8.0.4
pytimeparse==1.1.8
pytz==2025.2
PyWavelets==1.9.0
pywin32==311
PyYAML==6.0.3
referencing==0.37.0
regex==2026.1.15
requests==2.32.5
rich==14.3.2
rich-click==1.9.7
rpds-py==0.30.0
rsa==4.9.1
ruamel.yaml==0.19.1
scikit-learn==1.8.0
scipy==1.16.3
scmrepo==3.6.1
seaborn==0.13.2
semver==3.0.4
shap==0.50.0
shellingham==1.5.4
shortuuid==1.0.13
shtab==1.8.0
six==1.17.0
skops==0.13.0
slicer==0.0.8
smmap==5.0.2
sniffio==1.3.1
snowplow-tracker==1.1.0
SQLAlchemy==2.0.46
sqlparse==0.5.4
sqltrie==0.11.2
starlette==0.52.1
statsmodels==0.14.6
streamlit==1.54.0
sympy==1.14.0
tabulate==0.9.0
tenacity==9.1.4
text-unidecode==1.3
threadpoolctl==3.6.0
toml==0.10.2
tomlkit==0.14.0
toolz==1.1.0
torch==2.10.0
torchvision==0.25.0
tornado==6.5.4
tqdm==4.67.3
typeguard==4.5.0
typer==0.24.0
typing-inspect==0.9.0
typing-inspection==0.4.2
typing_extensions==4.15.0
tzdata==2025.3
tzlocal==5.3.1
ujson==5.11.0
urllib3==2.6.3
uuid6==2025.0.1
uvicorn==0.41.0
vine==5.1.0
visions==0.8.1
voluptuous==0.16.0
waitress==3.0.2
watchdog==6.0.0
watchfiles==1.1.1
wcwidth==0.6.0
websockets==16.0
Werkzeug==3.1.5
wordcloud==1.9.6
xgboost==3.2.0
yarl==1.22.0
ydata-profiling==4.18.1
zc.lockfile==4.0
zipp==3.23.0
